Gopkg.lock

```diff
@@ -898,7 +898,7 @@
   version = "v1.0"
 
 [[projects]]
-  digest = "1:67cd1819192a5fed8a7378065f27cc4cec3022c36b3f0d6fface973a788cb642"
```

Comment:
I think Gopkg.lock was commited by mistake, no ?

Reply:
Yes, I'm not sure why this changed.